database/mysql.go

```diff
@@ -136,6 +138,34 @@ func (r *TestMySQLDB) Close() error {
 	return nil
 }
 
+var SharedMySQL mySQLServer
```

Comment:
Since we know what the database config is (and the db config doesn't change from test to test), I don't think this needs to be a global variable. The `sync.Once` may still need to be a global var if we need it.